swagger.json

```diff
@@ -4350,6 +4432,21 @@
             "description": "ID of the asset",
             "type": "integer",
             "format": "int64"
+        "dataInputEndTimeParam": {
+            "name": "endMs",
+            "description": "Timestamp in milliseconds representing the end of the period to fetch, inclusive. Used in combination with startMs.",
+            "required": false,
+            "in": "query",
+            "type": "integer",
+            "format": "int64"
+        },
+        "dataInputStartTimeParam": {
+            "name": "startMs",
+            "description": "Timestamp in milliseconds representing the start of the period to fetch, inclusive. Used in combination with endMs.",
```

Comment:
We should also call out the default value.

Reply:
done!